src/webgpu/api/validation/render_pipeline/fragment_state.spec.ts

```diff
@@ -422,6 +432,6 @@ g.test('pipeline_output_targets,blend')
     const _success =
       info.color.type === sampleType &&
       componentCount >= kTexelRepresentationInfo[format].componentOrder.length &&
-      meetsExtraBlendingRequirement;
+      (meetsExtraBlendingRequirement || writeMask === 0);
```

Comment:
I think you are right. The "Otherwise..." is at the same level as 
> "If [get the entry point](https://gpuweb.github.io/gpuweb/#abstract-opdef-get-the-entry-point)([FRAGMENT](https://gpuweb.github.io/gpuweb/#dom-gpushaderstage-fragment), descriptor) has a [shader stage output](https://gpuweb.github.io/gpuweb/wgsl/#shader-stage-output) value output with [location](https://gpuweb.github.io/gpuweb/wgsl/#input-output-locations) attribute equal to index:"

Reply:
> I do believe we could loosen the validation in the way that you've done here, but it would require a spec change.

For reference the spec issue was https://github.com/gpuweb/gpuweb/issues/2013